src/main/java/com/iexec/sms/chain/WalletConfiguration.java

```diff
@@ -25,7 +25,7 @@
 @Configuration
 public class WalletConfiguration {
     @Bean
-    SignerService signerService(Web3jService web3jService, BlockchainConfig chainConfig) throws GeneralSecurityException {
+    SignerService signerService(Web3jService web3jService, ChainConfig chainConfig) throws GeneralSecurityException {
```

Comment:
Does it not work with `final` here ?
```suggestion
    SignerService signerService(final Web3jService web3jService, final ChainConfig chainConfig) throws GeneralSecurityException {
```